Sources/PointFree/Account/Account.swift

```diff
@@ -205,7 +215,7 @@ private let subscriptionPlanRows = View<Stripe.Subscription> { subscription in
                   `class`([Class.pf.components.button(color: .purple, size: .small)]),
                   href("#")
                   ],
-                  ["Upgrade"])
+                  ["Upgrade"]) // TODO: disable when subscription.status == .canceled
```

Comment:
We prob want a disabled style. And/or a `button(to:)` helper that knows when to be a button, when to be a link, when to be disabled, etc.